### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Java_Run and Debug_Usability : No any Dialogue/ audio description is provided for step by steps instructions shown in video. 

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6437]
### GitHub Tags
#A11ySev4; #A11yTCS; #A11yUsable; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023;

### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link and press enter.
3. Tab till "JAVA" link in the left navigation and press enter.
4. Tab till "Run and Debug" link present under the java and press enter.
5. Tab till Videos present on the page.
6. Observe that no any Dialogue/ audio description is provided for step by steps instructions shown in video. 
### Actual Result
No any Dialogue/ audio description is provided for step by steps instructions shown in video. 
### Note
Similar issue is observed throughout the application for the pre-recorded videos.

### Expected Result
The video should have a voiceover/ dialogue/ audio description where the instructor is talking about the steps they are taking in the video.

### User Impact
Visually challenged users will not able to get the information displayed in the video, so the dialogue/audio description should be provided to ensure these users will get the same information.

### Attachment
![Audio Description is not provided for the Pre-recorded videos on the page](https://github.com/microsoft/vscode-docs/assets/91586101/40cf315c-da92-40e4-b283-df33f143a755)

**Comment by manaswita-chougule** [comment]
@isidorn 
As per discussion with SME logged usability issue for the bug https://github.com/microsoft/vscode-docs/issues/6303#top

**Comment by gregvanl** [comment]
Closing old issue based on conversation in #6303
